airflow/providers/amazon/aws/triggers/base.py

```diff
@@ -102,9 +109,15 @@ def serialize(self) -> tuple[str, dict[str, Any]]:
             },
             **self.serialized_fields,
         )
+
+        # if we serialize the None value from this, it breaks subclasses that don't have it in their ctor.
         if self.region_name:
-            # if we serialize the None value from this, it breaks subclasses that don't have it in their ctor.
             params["region_name"] = self.region_name
+        if self.verify is not None:
+            params["verify"] = self.verify
+        if self.botocore_config is not None:
+            params["botocore_config"] = self.botocore_config
```

Comment:
nit, another way to do it.

```python
params.update(prune_dict({
  "region_name": self.region_name,
  "verify": self. verify,
  "botocore_config": self. botocore_config,
}))
```

Reply:
Yeah, makes sense. 

It is a bit change behaviour for `region_name` because right now `region_name=""` also excluded however I think it is edge case, because empty string it is invalid argument for boto3